pom.xml

```diff
@@ -43,6 +43,9 @@
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
 
+        <!-- bump version when performing releases -->
+        <geomesa.release.version>1.1.0-rc.6</geomesa.release.version>
```

Comment:
will the release plugin update the property for us? or will it be a manual step?